web/tests/i18n.test.js

```diff
@@ -109,12 +109,16 @@ run_test("tr_tag", ({mock_template}) => {
             realm_name_in_email_notifications_policy:
                 "Include organization name in subject of message notification emails",
             twenty_four_hour_time: "Time format",
+            dense_mode: "Dense mode",
             automatically_follow_topics_policy: "Automatically follow topics",
             automatically_unmute_topics_in_muted_streams_policy:
                 "Automatically unmute topics in muted channels",
             automatically_follow_topics_where_mentioned:
                 "Automatically follow topics where I'm mentioned",
```

Comment:
I'm not sure what the test means when it says testing the `tr_tag`. I'm not sure why `settings_tab` was chosen as the template. The `tr_tag` test was introduced in 82b5d9304b1b968f8be6f2d1e605360537eb3786.

Reply:
The test was written when "settings" was a small component not requiring a ton of mocking. I think probably best would be to replace it with a test rendering some very small template that uses a `tr` tag.